src/utils/common/constants.ts

```diff
@@ -68,8 +68,10 @@ export const constants = {
   sast: "sast",
   kics: "kics",
   sca: "sca",
+  scs: "sscs-secret-detection",
```

Comment:
make const of "sscs-secret-detection" if possible.

Reply:
Can not use "-" in const name so i used   scsSecretDetection: "sscs-secret-detection"